src/Ole32/Ole32.csproj

```diff
@@ -0,0 +1,9 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(DesktopAndPortableFrameworks)</TargetFrameworks>
```

Comment:
As far as I can see It should be `DesktopOnlyFrameworks` old32 doesn't seem supported on non-desktop no ?

Reply:
IIRC Ole32 is supported on Windows Phone, but I'm not quite sure if it's supported on, say, Windows Server Nano. Even on phone, I don't think that all Ole32 functions are available. I'm not quite sure what the delta between DesktopAndPortableFrameworks and DesktopOnlyFrameworks  is though. Are there other questions I could answer (or research the answers for) that would help us be sure that we are making the right choice?

Reply:
I think @AArnott will have the definitive answer on that.

My rule of thumb was that that Portable = UWP and so that it would never run on desktop.
(It also seem to be what our other project do)

But it's true that a netstandard dll could be run by dotnet core on windows and Ole32 WILL be there, but I don't think we support this case anywhere.

Ideally we would build our current "desktop" assemblies for everything including netstandard. And special-case the `uap` platform with what is currently our "portable" set.

But it's not what the current props file seem to do https://github.com/AArnott/pinvoke/blob/master/src/Directory.Build.props

Reply:
At least one method you added is MSDN documented as `Windows 2000 Professional [desktop apps only]`. So we should either target just Desktop profiles here, or carefully move the right subset of methods to a desktop-only source file as we've done in other projects (see Kernel32 as an example).

> But it's true that a netstandard dll could be run by dotnet core on windows and Ole32 WILL be there, but I don't think we support this case anywhere.

True. And if pressed upon by anyone, we could add the desktop-only APIs to the portable SKUs for the sake of dotnet.exe running on Windows. But we'd have to (as @vbfox said) compile `uap` to exclude it otherwise users would get store certification failures since it doesn't allow defining these desktop-only methods (even if unused). 
But for now, since no one is asking for it, the simplest and current policy is to make them only available when targeting desktop specifically. 

So yes, please change this to `$(DesktopOnlyFrameworks)`.

Reply:
Alright, done.